packages/stark-core/src/configuration/entities/application/app-config.entity.ts

```diff
@@ -1,11 +1,14 @@
+/* tslint:disable completed-docs*/
```

Comment:
The correct format is: `/* tslint:disable:completed-docs */`otherwise all rules are disabled ;)